weblate/static/loader-bootstrap.js

```diff
@@ -1449,6 +1449,22 @@ $(function () {
     (start, end, label) => {},
   );
 
+  /* Singular or plural new unit switcher */
+  $("input[name='new-unit-form-type']").on("change", function () {
+    const selected = $(this).val();
+    if (selected === "singular") {
+      $("input[name='new-unit-form-type']").removeAttr("checked");
+      $("#new-singular #show-singular").prop("checked", true);
+      $("#new-plural").addClass("hidden");
+      $("#new-singular").removeClass("hidden");
+    } else if (selected === "plural") {
+      $("input[name='new-unit-form-type']").removeAttr("checked");
+      $("#new-plural #show-plural").prop("checked", true);
+      $("#new-singular").addClass("hidden");
+      $("#new-plural").removeClass("hidden");
+    }
+  });
```

Comment:
Can this also sync forms content? So that the text doesn't disappear when you toggle?

Reply:
Yes, this can now transfer the inputs when toggling between the singular and plural forms.